src/main/java/uk/gov/hmcts/reform/civil/model/CourtLocation.java

```diff
@@ -4,15 +4,25 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Builder;
 import lombok.Data;
+import lombok.Setter;
+import uk.gov.hmcts.reform.civil.model.common.DynamicList;
+import uk.gov.hmcts.reform.civil.model.defaultjudgment.CaseLocation;
 
+@Setter
```

Comment:
shouldnt need a setter as you already have data, 
data is equivalent to getter and setter
docs:
`Equivalent to @Getter @Setter @RequiredArgsConstructor @ToString @EqualsAndHashCode.`

Reply:
cool, removed! @